.github/workflows/verify.yaml

```diff
@@ -403,8 +408,14 @@ jobs:
 
       - name: Data Plane Azure Data Factory Test
         uses: ./.github/actions/run-tests
+        env:
+          AZURE_TENANT_ID: ${{ secrets.AZURE_TENANT_ID }}
+          AZURE_CLIENT_ID: ${{ secrets.AZURE_CLIENT_ID }}
+          AZURE_CLIENT_SECRET: ${{ secrets.AZURE_CLIENT_SECRET }}
         with:
-          command: ./gradlew -p extensions/azure/data-plane/data-factory test -DincludeTags="AzureDataFactoryIntegrationTest"
+          command: |
+            ./gradlew -p extensions/azure/data-plane/data-factory test -DincludeTags="AzureDataFactoryIntegrationTest"
+            ./gradlew -p system-tests/azure-data-factory-tests test -DincludeTags="AzureDataFactoryIntegrationTest"
```

Comment:
Maybe it would be possible to use one command?
`./gradlew -p . -DincludeTags="AzureDataFactoryIntegrationTest"`
So that if someone adds AzureDataFactoryIntegrationTest in another module it's picked up by the workflow?

Reply:
Yes, you are right. It just that it takes longer as it needs to scan many modules and sub-modules. 

So, it definitely brings the advantage as you said but I was just thinking about the time it takes. But I will relook on this. Thanks